packages/cyberstorm/src/components/PackageCard.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/PackageCard.module.css";
+import { MetaItem } from "./MetaItem";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faDownload,
+  faThumbsUp,
+  faHardDrive,
+  faThumbtack,
+} from "@fortawesome/free-solid-svg-icons";
+import { Tag } from "./Tag";
+
+const defaultImageSrc = "";
+
+export interface PackageCardProps {
+  packageName: string;
+  description?: string;
+  imageSrc?: string;
+  downloadCount: string;
+  likes: string;
+  size: string;
+  author?: string;
+  link: string;
+  lastUpdated?: string;
+  isPinned?: boolean;
+  isNsfw?: boolean;
+  isDeprecated?: boolean;
+  colorScheme: "default";
+  categories: string[];
+}
+
+/**
+ * Cyberstorm PackageCard component
+ */
+export const PackageCard: React.FC<PackageCardProps> = (props) => {
+  const {
+    packageName,
+    description,
+    imageSrc,
+    downloadCount,
+    likes,
+    size,
+    author,
+    link,
+    lastUpdated,
+    isPinned,
+    isNsfw,
+    isDeprecated,
+    colorScheme,
+    categories,
+  } = props;
+
+  const authorLink = ""; //TODO: author link
+  //TODO: convert <a> tags into link components!
+  //TODO: Use LastUpdated component once one is developed
+
+  return (
+    <div className={`${styles.root} ${getStyle(colorScheme)}`}>
+      <a href={link} className={styles.imageWrapper}>
```

Comment:
Since this link doesn't have a text content it could use a `title`. This can be done once the anchors are replaced with the LinkProvider pattern as per @anttimaki's request.